var/spack/repos/spack_repo/builtin/packages/libxslt/package.py

```diff
@@ -41,22 +45,42 @@ class Libxslt(AutotoolsPackage):
     depends_on("c", type="build")
 
     depends_on("pkgconfig@0.9.0:", type="build")
+
     depends_on("iconv")
     depends_on("libxml2")
     depends_on("libxml2+python", when="+python")
     depends_on("xz")
     depends_on("zlib-api")
+
     depends_on("libgcrypt", when="+crypto")
+    conflicts("+crypto", when="platform=windows")
 
     depends_on("python+shared", when="+python")
     extends("python", when="+python")
 
+    # CMake was added in version 1.1.35
+    conflicts("build_system=cmake", when="@:1.1.34")
```

Comment:
Isn't this better done in the `build_system` as a `conditional`?

Reply:
Nice, yeah that is better. I hadn't used that before!